src/components/presentational/ValueSelector/ValueSelector.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { CSSProperties, useContext, useEffect, useState } from 'react';
+import './ValueSelector.css'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircle } from '@fortawesome/free-regular-svg-icons';
+import { faCheckCircle } from '@fortawesome/free-solid-svg-icons';
+import { ColorDefinition, resolveColor } from '../../../helpers/colors';
+import { LayoutContext } from '../Layout';
+
+export interface ValueSelectorProps {
+	title?: string;
+	titleColor?: ColorDefinition;
+	subtitle?: string;
+	subtitleColor?: ColorDefinition;
+	values: string[];
+	valueBackgroundColor?: ColorDefinition;
+	valueTextColor?: ColorDefinition;
+	checkboxColor?: ColorDefinition;
+	selectedValues?: string[];
+	selectedButtonBackgroundColor?: ColorDefinition;
+	selectedButtonTextColor?: ColorDefinition;
+	selectedCheckboxColor?: ColorDefinition;
+	onChange?: (selectedValues: string[]) => void;
+	variant?: 'default' | 'checkboxes';
+	multiSelect?: boolean;
+	preventEmptySelections?: boolean;
+	innerRef?: React.Ref<HTMLDivElement>;
+}
+
+export default function (props: ValueSelectorProps) {
+	const layoutContext = useContext(LayoutContext);
+
+	const [selectedValues, setSelectedValues] = useState<string[]>([]);
```

Comment:
Why have a state variable here?  Seems like this is pretty similar to the SegmentedControl where it would make sense to be stateless?  Especially since selectedValues is a parameter.

Whatever parent component you use is presumably gonna need to save the value anyways, no?

Reply:
I think I added it because it felt weird for the component to not be functional in the story book.  However, you are correct, there should always be a containing component that is managing that state.

I've removed the state from the component and updated the story rendering to maintain that state so that component remains functional in the storybook.